src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -35,7 +35,7 @@ public static Person[] getSamplePersons() {
                        new Address("Blk 30 Geylang Street 29,#06-40"),
                        new DateOfBirth("1990-01-01"),
                        new Gender("M"),
-                       getTagSet("friends"),
+                       getAllergySet("friends"),
```

Comment:
Maybe replace friends with something else like peanuts, since tag has been changed to allergies